tests/test_lag_2.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+
+import time
+import logging
+import os
+
+from ptf_runner import ptf_runner
+
+@pytest.fixture(scope="module")
+def common_setup_teardown(duthost, ptfhost, testbed, conn_graph_facts):
+    logging.info("########### Setup for lag testing ###########")
+
+    lag_facts = duthost.lag_facts(host = duthost.hostname)['ansible_facts']['lag_facts']
+
+    if lag_facts['names'] == []:
+        pytest.skip("No lag configuration found in %s" % duthost.hostname)
+
+    mg_facts = duthost.minigraph_facts(host=duthost.hostname)['ansible_facts']
+    fanout_neighbors = conn_graph_facts['device_conn']
+    vm_neighbors = mg_facts['minigraph_neighbors']
+
+    # Copy PTF test into PTF-docker for test LACP DU
+    test_files = ['lag_test.py', 'acs_base_test.py', 'router_utils.py']
+    for test_file in test_files:
+        src = "roles/test/files/acstests/%s" % test_file
+        dst = "/tmp/%s" % test_file
+        ptfhost.copy(src=src, dest=dst)
+
+    # Copy tests to the PTF-docker
+    ptfhost.copy(src="ptftests", dest="/root")
+
+    # Inlucde testbed topology configuration
+    testbed_type = testbed['topo']['name']
+    support_testbed_types = frozenset(['t1-lag', 't0', 't0-116'])
+    if testbed_type in support_testbed_types:
```

Comment:
Looks like it is unnecessary to load those vars because nobody uses them.
I will keep this TODO and decide if keep the logic here after final test.